### DISCUSSION TITLE
How to display VDH v9 new interface in another language than the browser's one?

### DISCUSSION THREAD

**Discussion by Patriccollu** [aclap-dev/video-downloadhelper#881] [Language & Translation]
Hello,

I'm Corsican translator for VDH and I use a Windows 10 French system with Firefox and Chrome browsers in French because, as you may know, **Corsican language is not yet available**, neither on Firefox and Chrome browsers, nor as Windows system language.

With the v8 User Interface, it was possible to use another language in VDH which was different to the one used by the browsers, by using the _Import_ function in _Translation_ tab.

I installed the v9.0.2.3a1 beta version on Firefox and noticed the new UI was French (as the browser's language). I tried _Import Settings_ function from the new UI using an existing Corsican `messages.json` language file as input but didn't see any changes. So I switched to the legacy UI, imported the same Corsican file and saved it, which successfully displayed the UI in Corsican. But as soon as I switched back to the New UI, the language reverted to French!

What is the solution to display the new VDH v9 interface in another language than the browser language?

Best regards,
Patriccollu.

**Comment by paulrouget** [comment]
Ah, nope, no way sadly. I'll try to fix that!

**Comment by Patriccollu** [comment]
@paulrouget 
Hi Paul,

We would really appreciate if you could fix that!

Since VDH has been translated in Corsican three years ago (July, 2021 with version 7.6.1a1) - and as long as Corsican language is not recognized by the browsers - this is the only way to use VDH in Corsican!

Thanks for your efforts,
Patriccollu.

**Comment by Patriccollu** [comment]
@paulrouget 
Hi Paul,

Any news on this?

Thanks for your efforts,
Patriccollu.

**Comment by Patriccollu** [comment]
@paulrouget 
Hi Paul,

I hope that you are well and I wish you - and the whole team - a Happy New Year!

Have you had a chance to see how to solve this problem with v9 new interface?

Many thanks,
Patriccollu.

**Comment by Patriccollu** [comment]
@paulrouget 
Hi Paul,

Have you had a chance to fix this problem with v9 new interface?

Thanks for your help,
Patriccollu.

**Reply by jcfrog** [reply]
Sorry, I'm afraid Paul didn't have time for this in the last weeks.

**Reply by Patriccollu** [reply]
@jcfrog Thanks for your very responsive answer!

As I explained some time ago to Paul - and as long as Corsican language is not recognized by the browsers - legacy UI is currently the only way to use VDH in Corsican!

We will wait :-(

**Comment by Patriccollu** [comment]
@paulrouget
Hi Paul,

Please can you tell us when you will fix this problem with v9 new interface?

Thanks for your help,
Patriccollu.

**Comment by Patriccollu** [comment]
Hello,

>  @paulrouget on Jul 16, 2024	Maintainer 
> Ah, nope, no way sadly. I'll try to fix that!

I can understand that this problem does not affect a lot of people as it concerns Corsican and eventually all languages not already taken into account by the browsers.

Nevertheless, is there a chance to find a suitable solution to this problem?

Have you had some ideas to keep on working on it?

Thanks a lot for your help,
Patriccollu.

**Reply by garry-ut99** [reply]
> Patriccollu: Nevertheless, is there a chance to find a suitable solution to this problem?
> Patriccollu: Have you had some ideas to keep on working on it?

It's unclear whether you are asking about VHD v9 only or about any version (means V10 too), but as for VDH v9 - most likely not, as it's not supported anymore, means: abandoned.

**Comment by Patriccollu** [comment]
Hi @garry-ut99,

Thank you for your help.

When I opened this issue, VDH V10 didn't exist and V9 was the latest version but, unfortunately, the problem also concerns V10!

Actually it occurs on all versions since V9 as long as VDH doesn't propose an option to **Import a Translation** which could be different than the language used by the browser.

For your information, I translated VDH in Corsican in July 2021 for version 7.6.1a1 then keeping on translating all V8 releases. At that time it was possible to **Import (and Export) the translation to/from a browser**. When V9 was released, I translated all the new strings but the new UI interface was not available in Corsican; only the Legacy UI was. That's why this issue was opened.

Any help is welcome.

Regards,
Patriccollu.

**Comment by Patriccollu** [comment]
Hello @paulrouget , do you have some news on this?
